app/build.gradle.kts

```diff
@@ -1,196 +1,36 @@
 import com.google.gms.googleservices.GoogleServicesPlugin.GoogleServicesPluginConfig
 
 plugins {
-    alias(libs.plugins.android.application)
-    alias(libs.plugins.kotlin.android)
-    alias(libs.plugins.ksp)
-    alias(libs.plugins.kotlin.parcelize)
+    alias(libs.plugins.homeassistant.android.application)
     alias(libs.plugins.firebase.appdistribution)
     alias(libs.plugins.google.services)
-    alias(libs.plugins.hilt)
-    alias(libs.plugins.compose.compiler)
+    alias(libs.plugins.homeassistant.android.dependencies)
 }
 
 android {
-    namespace = "io.homeassistant.companion.android"
-
-    compileSdk = libs.versions.androidSdk.compile.get().toInt()
-
-    ndkVersion = "21.3.6528147"
-
     useLibrary("android.car")
```

Comment:
Does it make sense to have this also for the `:app`?

Reply:
Currently I think yes because of it's use in `BaseActivity` and `BaseVehicleScreen`? Otherwise the import breaks (unresolved references).

Reply:
I'm going to keep this in my notes for later :)